sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md

```diff
@@ -78,6 +78,17 @@ title: QuestionAnsweringAuthoringClient
 
 ### General customizations
 
+#### Add docs to authoring operations
+
+```yaml
+directive:
+- from: questionanswering-authoring.json
+  where: $.paths.*.*
```

Comment:
Depending on how you pull autorest, you might actually try `where-operation-match: /.*/`. It didn't work for .NET because we're using an older version. This works too, though.